### ISSUE TITLE
Action items from Accessibility sync

### ISSUE THREAD

**Issue by beccamc** [dotnet/machinelearning-modelbuilder#973]
- [x] Focus should be on the first interactable element (e.g. CPU Environment tab, file picker, input time box)
- [x] Make sure to have the dotted line to indicate focus. Also ensure the focus line has correct visibility/contrast (black dotted line on dark blue background doesn’t work)
- [x] Use the AutomationProperties HelpText to add additional tooltip information. 
- [ ] Local/Azure tag on Scenarios shouldn’t be accessible. We should look at what we want read when users navigate to a scenario card. The title + description? Title + tag? Take a look at what New Project Dialog does since they also have tags.
- [x] Look into the async adding of projects. If both “Add projects” and “Next steps” button are disabled it is disconcerting to the user. If it’s less than one second its probably fine, if not we may want to add some text here or progress indicator, would be useful for sighted users as well. Speak with Ryan Shugart for more info. 
- [x] Speak with Guy Barker for more info on if our Data page data table is correct.

**Comment by beccamc** [comment]
Page focused fixed with https://github.com/dotnet/machinelearning-tools/pull/747

**Comment by beccamc** [comment]
These items have been addressed.
